user-impl/src/test/java/com/example/auction/user/impl/UserRepositoryTest.java

```diff
@@ -87,11 +91,13 @@ public void shouldPaginateUserRetrieval() throws InterruptedException, Execution
             feed(new PUserEvent.PUserCreated(buildFixture(createdAt, i)));
         }
 
+
         PaginatedSequence<User> createdUsers = Await.result(userRepository.getUsers(1, 10));
-        assertEquals(25, createdUsers.getCount());
+        assertEquals(initialCount + size, createdUsers.getCount());
+
         assertEquals(10, createdUsers.getItems().size());
         // default ordering is time DESC so page 2 of size 10 over a set of 25 returns item ids 5-14. On that seq, the fifth item is id=10
-        assertEquals("user10", createdUsers.getItems().get(4).getName());
+        assertEquals("admin4", createdUsers.getItems().get(4).getName());
```

Comment:
It seems that I was wrong about this one. I'm getting random values here. 
I've seen the following values so far:
`admin4`
`admin10`
`admin22`
`admin18`